.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You've added this stuff into the .gitignore but haven't cleared the git cache. Because it was already committed it still remains (which obviously we don't want). Have a look at http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/
Please let me know if you have any trouble/questions about this.